lib/src/interceptors/http_interceptor.dart

```diff
@@ -1 +1,56 @@
-// TODO: HttpInterceptor
+import 'dart:io';
+import 'package:dio/dio.dart';
+import 'package:http_mock_adapter/src/handlers/request_handler.dart';
+import 'package:http_mock_adapter/src/history.dart';
+import 'package:http_mock_adapter/src/request.dart';
+import 'package:http_mock_adapter/src/adapter_interface.dart';
+
+class MainInterceptor extends Interceptor
```

Comment:
I think `MainInterceptor` is a horrible name in our context.

I would prefer it to be `MockInterceptor` or something similar. Any ideas? @campfire5 @nataliehippe  @LukaGiorgadze

Reply:
DioInterceptor